homeassistant/components/plex/media_player.py

```diff
@@ -544,6 +545,15 @@ def device_info(self):
         if self.machine_identifier is None:
             return None
 
+        if self.device_product in TRANSIENT_DEVICE_MODELS:
```

Comment:
Shouldn't we just return None?

Reply:
I originally wanted to delete the current devices, but that also removed the associated entities. This moves them to a new service device instead so the old device can get safely removed by the new orphan cleanup code.

Reply:
BTW, I think I'd actually prefer to not have any devices attached to these entities, but returning `None` doesn't disassociate them from existing devices.